src/python/WMCore/JobSplitting/JobFactory.py

```diff
@@ -420,3 +419,26 @@ def getPerformanceParameters(self, defaultParams):
         sizePerEvent = defaultParams.get('sizePerEvent', None) or 0
         memory = defaultParams.get('memoryRequirement', None) or 0
         return timePerEvent, sizePerEvent, memory
+    
+    def checkCreateFailConditoin(self, failedJob, failedReason):
+
+        valueMap = {"MaxWallTimeHours": "estimatedJobTime", 
+                    "MaxMemory": "estimatedMemoryUsage", 
```

Comment:
Alan, I just change the unit to more human configurable. (I don't think they want to set by minutes)
But if you think it is more clear to use the same name and unit I will change that in the config. (and add comments which unit we are using)

It seems we are setting jdl key request_disk and request_memory instead of RequestDisk, Is it a bug?
Should it be RequestDisk and RequestMemory? Are both units KB?
https://github.com/dmwm/WMCore/blob/master/src/python/WMCore/BossAir/Plugins/CondorPlugin.py#L1025